etherpad/docker-compose.yml

```
@@ -0,0 +1,67 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: etherpad_app_1
+      APP_PORT: 9001
+      
+  app:
+    image: etherpad/etherpad:2.2.7@sha256:980f227a17569901aaa8f98d8bb6eff51e15c04ad3e2ea0a1692662adbcc2090
+    tty: true
+    stdin_open: true
+    volumes:
+      - ${APP_DATA_DIR}/data/plugins:/opt/etherpad-lite/src/plugin_packages
+      - ${APP_DATA_DIR}/data/etherpad-var:/opt/etherpad-lite/var:rw
+    depends_on:
+      - postgres
+    environment:
+      NODE_ENV: production
+      ADMIN_PASSWORD: umbrel
+      DB_CHARSET: utf8mb4
+      DB_HOST: postgres
+      DB_NAME: etherpad
+      DB_PASS: admin
+      DB_PORT: 5432
+      DB_TYPE: "postgres"
+      DB_USER: admin
+      DEFAULT_PAD_TEXT: 
+      # The env var DEFAULT_PAD_TEXT cannot be unset or empty
+      DISABLE_IP_LOGGING: "false"
+      LOGLEVEL: DEBUG
+      SOFFICE: null
+      ALLOW_UNKNOWN_FILE_ENDS: "true"
+      TRUST_PROXY: "true"
+      SKIN_NAME: colibris
+      # Either no-skin, colibris or an existing directory under src/static/skins
+      SKIN_VARIANTS: super-light-toolbar super-light-editor light-background   
+      # The Colibris skin allows you to customize the appearance of three containers: toolbar, editor, and background. 
+      # You can choose from four color combinations (super-light, light, dark, super-dark) for each container. 
+      # For example, using "dark-toolbar" sets the toolbar to dark. 
+      # Multiple variants can be applied, and by default, the skin variant is "super-light-toolbar super-light-editor light-background." 
+      # You can also make the editor full-width by adding the "full-width-editor" variant.
+      SHOW_SETTINGS_IN_ADMIN_PAGE: "true"
+      SUPPRESS_ERRORS_IN_PAD_TEXT: "false"   
+    restart: on-failure
+    user: "1000:1000"
+
+  postgres:

```

Run postgres as non-root user.
```suggestion
  postgres:
    user: "1000:1000"
```